src/networking.c

```diff
@@ -3239,7 +3239,7 @@ int isClientConnIpV6(client *c) {
 
 /* Concatenate a string representing the state of a client in a human
  * readable format, into the sds string 's'. */
-sds catClientInfoString(sds s, client *client) {
+sds catClientInfoString(sds s, client *client, int mask_sensitive) {
```

Comment:
nit. lets try and stick to the current naming eg hide_user_data
```suggestion
sds catClientInfoString(sds s, client *client, int hide_user_data) {
```